echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/PipelineCache.java

```diff
@@ -132,6 +145,40 @@ void pollPipelineConfigs() {
     }
   }
 
+  private List<Pipeline> fetchHydratedPipelines() {
+    List<Map> rawPipelines = front50.getPipelines();
+    if (rawPipelines == null) {
+      return Collections.EMPTY_LIST;
+    }
+
+    Predicate<Map> isV2Pipeline = p -> {
+      return p.getOrDefault("type", "").equals("templatedPipeline") &&
+        p.getOrDefault("schema", "").equals("v2");
+    };
+
+    return rawPipelines.stream()
+      .filter(Objects::nonNull)
+      .map(p -> {
+        if (isV2Pipeline.test(p)) {
+          try {
+            Map plannedPipeline = orca.v2Plan(p); // Catch in a local variable so we can increment metrics.
+            registry.counter("orca.requests").increment();
```

Comment:
Upon closer inspection of our existing metrics ... this isn't actually necessary, we've already got `okhttp.requests` tracking inter-service success and failures. 

This isn't adding anything on top of that so could just kill it (I was otherwise going to suggest one planning-specific metric that had a tag indicating success vs failure, rather than two metrics).

Reply:
Done.